vendor/mlir/mlir.BUILD

```diff
@@ -39,7 +39,7 @@ cc_binary(
     srcs = glob([
         "tools/mlir-tblgen/*.cpp",
         "tools/mlir-tblgen/*.h",
-    ]),
+    ]) + ["@com_intel_plaidml//pmlc/dialect/op_lib:gen_op_lib_wrappers"],
```

Comment:
We should make the `tool` attribute of the `mlir_tblgen` allow a user-specified label. We shouldn't be pulling in `com_intel_plaidml` from MLIR's perspective.